controller/indexUser.js

```diff
@@ -60,10 +89,103 @@ const registration = async (req, res, next) => {
     next(error)
   }
 }
+const verificationToken = async (req, res, next) => {
+  const {verificationToken} = req.params
+  const user = await User.findOne({verificationToken})
+  console.log(user)
 
+  if (user) {
+      if (user.verify) {
+    return res.status(404).json({
+      status:"Bad request",
+      code: 400,
+        data:{
+        message:" Verification has already been passed"
+        },
+  })
+}
+    const updateUser = await User.findOneAndUpdate({verificationToken}, {verify:true, verificationToken:null})    
+    res.status(200).json({
+      status: 'succes',
+      code: 200,
+      data: {
+        message:'Verification successful'
+      }
+    })
+  }
+      else {
+    res.status(404).json({
+      status: 'Not Found',
+      code: 404,
+      data: {
+        message:'User not found'
+      }
+    })
+    }
+}
+const resendingEmail = async (req, res, next) => {
+  const { email } = req.body;
+  const user = await User.findOne({ email })
+  if (!email) {
+    return res.status(400).json({
+      status: "Bad request",
+      code: 400,
+      data: {
+         message:"missing required field email"
+      }
+    })
+  }
+  if (!user) {
+    return res.status(404).json({
+      status: "Not found",
+      code: 404,
+      data: {
+        message:"Please registration"
+      }
+    })
+  }
+  if (user.verify==true) {
```

Comment:
zawsze uzywaj `===`. Natomiast tutaj wystarczy `user.verify`, bo `verify` to Bool. Nie ma potrzeby robic `true === true` :)